oidc-ui/src/components/FormAction.js

```diff
@@ -16,12 +16,7 @@ export default function FormAction({
         <button
           type={type}
           value={type}
-          className={
-            className +
-            (disabled
-              ? "text-slate-400 bg-white"
-              : "text-white bg-gradient-to-t from-cyan-500 to-blue-500 hover:bg-gradient-to-b")
-          }
+          className={className + " primary-button"}
```

Comment:
does the primary-button class handle the disabled state as well?

Reply:
yes @anshulv1401